Week-2/Session-4/SocialNetworkForm.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Social Network Form</title>
+    <style>
+        body {
+            position: relative;
+        }
+        /* Why do i have to define this ?
+        Without this css block my absolute positioning doesn't work */
+        .input-row {
+            display: flex;
+            margin-bottom: 10px;
+        }
+         .textarea-row {
+            display: flex;
+            margin-bottom: 45px;
+        }
+        input {
+            position: absolute;
+            margin-left: 150px;
```

Comment:
Same. Try using the gap property to space out flex child elements. This is a much more responsive behaviour.

Reply:
Done